### PR TITLE
TF-933 [PART-2] Apply new design for search 

### PR DESCRIPTION
**Pull request by dab246**
#933 

### Resolved
- [x] Create new search view for Mobile/Tablet

https://user-images.githubusercontent.com/80730648/190505825-9c8a824b-9251-4d35-abc2-d78ef7835394.mp4

https://user-images.githubusercontent.com/80730648/190505930-cb567024-d4ad-4e2a-b305-e1ed148e66ae.mp4

### CODE REVIEW

**Review by ManhNTX — APPROVED** [review]

File: lib/features/thread/presentation/mixin/base_email_action_controller.dart [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+import 'package:tmail_ui_user/features/mailbox/presentation/model/mailbox_actions.dart';
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart';
+import 'package:tmail_ui_user/features/thread/presentation/model/delete_action_type.dart';
+import 'package:tmail_ui_user/main/localizations/app_localizations.dart';
+import 'package:tmail_ui_user/main/routes/app_routes.dart';
+import 'package:tmail_ui_user/main/routes/route_navigation.dart';
+
+mixin BaseEmailActionController {
+
+  final mailboxDashBoardController = Get.find<MailboxDashBoardController>();
```
Comment (hoangdat) [inline lib/features/thread/presentation/mixin/base_email_action_controller.dart:30]: this object is stick to `MailboxDashBoardController`. So IMO, the name should be `EmailActionController`

Comment (dab246) [reply lib/features/thread/presentation/mixin/base_email_action_controller.dart:30]: oke, i update it.

**Comment by dab246** [comment]
@hoangdat Please review again

**Review by hoangdat — APPROVED** [review]

**Comment by hoangdat** [comment]
please rebase

**Pull request merged**